db.js

```diff
@@ -0,0 +1,33 @@
+const db_File_Name = "tasksFile.JSON"
+fs = require('fs')
+
+function ReadData(){
+    if (fs.existsSync(db_File_Name))
+    {
+        return fs.readFileSync(db_File_Name)
+    }
+    else
+    {
+        initial_data = "{}"
+        fs.writeFileSync(db_File_Name, initial_data)
+        return initial_data
+    }
+}
+
+function WriteData(data){
```

Comment:
use javascript/nodejs conventin, for example the name of the function should start with small letter:
writeData and not WriteData

read JavaScript convention and use the right convention in all the projects (for example you can remove spaces between lines)